driver/src/contract.rs

```diff
@@ -161,10 +161,10 @@ impl SnappContract for SnappContractImpl {
                 "applyWithdrawals",
                 (slot, merkle_root, prev_state, new_state, withdraw_hash),
                 account,    
-                Options::with(|opt| { // usual gas estimate is not working
-            opt.gas_price = Some(25.into());
-            opt.gas = Some(1_000_000.into());
-        }),
+                Options::with(|mut opt| { // usual gas estimate is not working
```

Comment:
Hm, not sure what is happening under the hood. Maybe your IDE sees the next line as changes of the object opt. But in the end it is only initialization and this is why the compiler is not asking for it.

I am only guessing here. I am curious on Felixs answer

Reply:
We are mutating opt, so it should be market as mut. I'm surprised it compiled before. I think technically, it should even be `&mut` (https://tomusdrw.github.io/rust-web3/web3/contract/struct.Options.html#method.with)

Reply:
I will change it to &mut

Reply:
@fleupold &mut gave errors! I will not change it.